recipes/mender-monitor/debian-master/mender-monitor.install

```diff
@@ -0,0 +1,27 @@
+/usr/bin/mender-monitord
+/usr/bin/mender-monitorctl
+
+/etc/mender-monitor/monitor.d/service.sh
+/etc/mender-monitor/monitor.d/log.sh
+/etc/mender-monitor/monitor.d/dbus.sh
```

Comment:
Where are the new demo subsystems?

Reply:
Only part of the demo package install

Reply:
Exactly, but I don't see them there in `mender-monitor-demo.install`

Reply:
Have another look :smile_cat:

Reply:
You added them now in the last force-push. Good :+1: